src/doc/guide.md

```diff
@@ -506,9 +503,9 @@ src/main.rs:2     let x: int;
                       ^
 ```
 
-Rust warns us that we never use the variable binding, but since we never use it,
-no harm, no foul. Things change if we try to actually use this `x`, however. Let's
-do that. Change your program to look like this:
+Rust warns us that we never use the variable, but since we never use it, no
```

Comment:
As above and (maybe?) in previous convos, variable bindings are relationships, not things you can point to in code listings. This usage is peculiar and idiosyncratic.

Of course I don't want to step on toes if this is the desired language. I'll share my reasoning in depth if needed, and we can get this cleared up!